### PR TITLE
Vloginov/add polynomials handwritten

### PR DESCRIPTION
**Pull request by morkovka1337**
Add model for recognizing handwritten polynomials

### CODE REVIEW

**Comment by eaidova** [comment]
Jenkins ok to test

**Review by eaidova — APPROVED** [review]

File: models/intel/formula-recognition-polynomials-handwritten-0001/description/formula-recognition-polynomials-handwritten-0001.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+|-----------------------------------------------|-----------|
+| GFlops                                        | 12.8447   |
+| MParams                                       | 8.6838    |
+
+
+### Performance
+
+### Inputs
+
+1.	Name: `imgs` , shape: [1x3x100x990]. An input image in the [1xCxHxW] format.
```
Comment (vladimir-dudnik) [inline models/intel/formula-recognition-polynomials-handwritten-0001/description/formula-recognition-polynomials-handwritten-0001.md:42]:
```suggestion
1.	Name: `imgs` , shape: [1x3x96x990]. An input image in the [1xCxHxW] format.
```

File: models/intel/formula-recognition-polynomials-handwritten-0001/description/formula-recognition-polynomials-handwritten-0001.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+    The expected channel order is BGR.
+
+### Outputs
+1.	Name: `hidden`, shape: [1x512]. Initial context state of the LSTM cell.
+2.	Name: `context`, shape: [1x512]. Initial hidden state of the LSTM cell.
+3.	Name: `init_0`, shape: [1x256]. Initial state of the decoder.
+5.	Name: `row_enc_out`, shape: [1x7x62x512]. Features from encoder that are fed to a decoder.
```
Comment (vladimir-dudnik) [inline models/intel/formula-recognition-polynomials-handwritten-0001/description/formula-recognition-polynomials-handwritten-0001.md:49]:
```suggestion
4.	Name: `row_enc_out`, shape: [1x7x62x512]. Features from encoder that are fed to a decoder.
```

File: models/intel/formula-recognition-polynomials-handwritten-0001/description/formula-recognition-polynomials-handwritten-0001.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+3.	Name: `output_prev`, shape: [1x256]. Current state of the decoder.
+4.	Name: `row_enc_out`, shape: [1x7x62x512]. Encoded features.
+5.	Name: `tgt`, shape: [1x1]. Index of the previous symbol.
+
+### Outputs
+
+1.	Name: `dec_st_c` , shape: [1x512]. Current context state of the LSTM cell.
+2.	Name: `dec_st_h` , shape: [1x512]. Current hidden state of the LSTM cell.
+3.	Name: `output`, shape: [1x256]. Current state of the decoder.
+1.	Name: `logit`, shape: [1xVocab_Size]. Classification confidence scores in the [0, 1] range
```
Comment (vladimir-dudnik) [inline models/intel/formula-recognition-polynomials-handwritten-0001/description/formula-recognition-polynomials-handwritten-0001.md:79]:
```suggestion
4.	Name: `logit`, shape: [1xVocab_Size]. Classification confidence scores in the [0, 1] range
```

File: models/intel/formula-recognition-polynomials-handwritten-0001/description/formula-recognition-polynomials-handwritten-0001.md [hunk 4]
```diff
@@ -0,0 +1,84 @@
+# formula-recognition-polynomials-handwritten-0001 (composite)
+
+## Use Case and High-Level Description
+
+This is an im2latex composite model that recognizes latex formulas.
+The model uses vocabulary file to predict sequence of latex tokens.
+The model is built on the resnext50 backbone with additional attention-based text recognition head.
```
Comment (vladimir-dudnik) [inline models/intel/formula-recognition-polynomials-handwritten-0001/description/formula-recognition-polynomials-handwritten-0001.md:7]: would be nice to add what dataset model was trained on. If that is internal dataset then just say trained on internal dataset, containing handwritten symbols

Comment (morkovka1337) [reply models/intel/formula-recognition-polynomials-handwritten-0001/description/formula-recognition-polynomials-handwritten-0001.md:7]: Thank you for your suggestion, the description was updated.

File: models/intel/formula-recognition-polynomials-handwritten-0001/description/formula-recognition-polynomials-handwritten-0001.md [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+3.	Name: `output_prev`, shape: [1x256]. Current state of the decoder.
+4.	Name: `row_enc_out`, shape: [1x6x62x512]. Encoded features.
+5.	Name: `tgt`, shape: [1x1]. Index of the previous symbol.
+
+### Outputs
+
+1.	Name: `dec_st_c` , shape: [1x512]. Current context state of the LSTM cell.
+2.	Name: `dec_st_h` , shape: [1x512]. Current hidden state of the LSTM cell.
+3.	Name: `output`, shape: [1x256]. Current state of the decoder.
+4.	Name: `logit`, shape: [1xVocab_Size]. Classification confidence scores in the [0, 1] range
```
Comment (eizamaliev) [inline models/intel/formula-recognition-polynomials-handwritten-0001/description/formula-recognition-polynomials-handwritten-0001.md:81]:
```suggestion
4.	Name: `logit`, shape: [1xN], where `N` is a vocabulary size. Classification confidence scores in the [0, 1] range
```

**Review by eizamaliev — CHANGES_REQUESTED** [review]

**Review by vladimir-dudnik — APPROVED** [review]

**Review by eizamaliev — APPROVED** [review]

**Pull request merged**